app/services/project_data_service.rb

```diff
@@ -0,0 +1,90 @@
+require "base64"
+
+class ProjectDataService
+
+  def initialize(project, access_token)
+    @project = project
+    @access_token = access_token
+    @client = Octokit::Client.new(access_token: access_token)
+    @circle_config = circle_config
+  end
+
+  def github_org
+    Organization.find(@project.organization_id).name
```

Comment:
This can just be `@project.organization.name`.

Related: this isn't the only place we assume an `Organization` name matches a Github org name. We also do it here: https://github.com/artsy/horizon/blob/88dabc801c01d1bc58ebe4b8d1b9f9911063d444/app/models/deploy_strategy.rb#L21

It might be good to encapsulate that somewhere like `project.github_org_name` (and use the new method in both places) so we have the option to improve it in the future (like by basing it on the actual git remote).